### DISCUSSION TITLE
There someway to we notify when break time is over ?

### DISCUSSION THREAD

**Discussion by Thiteago** [ErnaneJ/tradingworks-plus#22] [Q&A] [answered]
I'm thinking about that, if there someway to  calculate this with some function we already have

**Comment by ErnaneJ** [comment, marked as answer]
Yes my friend, I've been thinking about that too.

We currently have a small problem with the way notifications are sent. Especially when it comes to the end of the break. What happens is that as, when leaving the interval, the value present in the variable that stores that time never changes, every minute the verification for sending will show as true that it is to be sent.

However, I've been thinking of a way around this. We could use the local storage system to store some specific ID for each message sent and always, before sending a new one, we check if it is already marked as sent or not.

The only extra concern will be that at the end of each working day we need to clear these ids so that it is possible to send them the next day. In this way, the ID could be related to the current day (considering that the user will only have one journey per workday and that these messages should only be sent once per shift).

I'll create a new branch to build on this and we can further discuss the implementation. right?

Thanks!

**Reply by Thiteago** [reply]
Oh sure, that local storage idea is very interesting and i think there is any performatic problem with that, i will wait the start of this implementation , thank you !

**Reply by ErnaneJ** [reply]
Exactly @Thiteago! I'm studying the implementation better. You can follow along and contribute to this in this [branch](https://github.com/ErnaneJ/tradingworks-plus/tree/refactor-sending-messages).

**Reply by ErnaneJ** [reply]
Hello @Thiteago, I just opened a PR ([#26](https://github.com/ErnaneJ/tradingworks-plus/pull/26)) about this.
